lib/Backends/Interpreter/InterpreterNodes.cpp

```diff
@@ -1026,8 +1026,7 @@ void BoundInterpreterFunction::fwdBatchNormalizationFloatImpl(
   auto biasH = getWeightHandle<ElemTy>(I->getBias());
   auto meanH = getWeightHandle<ElemTy>(I->getMean());
   auto varH = getWeightHandle<ElemTy>(I->getVar());
-  unsigned_t channelIdx =
-      I->getChannelIdx(); // NOTE: We only support NTHWC, NHWC, NWC and NCW
+  unsigned_t channelIdx = I->getChannelIdx();
```

Comment:
e.g. here

Reply:
@jfix71, Before rebasing, the last commit was #5195, now pulled in the latest commit #5300 and rebased on top of it.
Let me know if there are still any more conflicts.